modules/cluster-logging-collector-log-forward-logs-from-application-pods.adoc

```diff
@@ -0,0 +1,63 @@
+[id="cluster-logging-collector-log-forward-logs-from-application-pods_{context}"]
+= Forwarding application logs from specific pods
+
+As a cluster administrator, you can use Kubernetes pod labels to gather log data from specific pods and forward it to a log collector.
+
+Suppose that you have an application composed of pods running alongside other pods in various namespaces. If those pods have labels that identify the application, you can gather their log data and output it to a specific log collector.
+
+You can also combine namespace and pod labels to restrict the gathering of log data to pods in specific namespaces.
+
+To specify pod labels this way, you create a `ClusterLogForwarder` custom resource (CR) that has one or more of the following elements:
+
+* An input that specifies the pod labels
+* An output that forwards log data to a log collector
+* A pipeline that connects the input to the output
+
+.Procedure
+
+. Create a `ClusterLogForwarder` custom resource (CR) YAML file.
+
+. In the YAML file, specify the pod labels using simple equality-based selectors under `inputs[].name.application.selector.matchLabels`, as shown in the following example.
+
+. Optional: Restrict the gathering of log data to specific namespaces by using `inputs[].name.application.namespaces`, as shown in the following example.
```

Comment:
- Since this is an optional step, also covered in the 6th call out, can we move the example under the step on line 20? Maybe then reword this part to: as shown in the example.
- Since it is mentioned in the 6th call out, do we need this step at all? If it is worth highlighting to the end users, then yes, we should retain it.
- Add a '+' above and below the example so that all the steps are rendered appropriately. Currently, the last step- Create the CR object,  is not being rendered in the appropriate manner: https://deploy-preview-32334--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-external.html#cluster-logging-collector-log-forward-logs-from-application-pods_cluster-logging-external